lib/smile-identity-core/web_api.rb

```diff
@@ -91,8 +91,13 @@ def id_info=(id_info)
       # if it's a boolean
       updated_id_info[:entered] = id_info[:entered].to_s if !updated_id_info[:entered].nil? == updated_id_info[:entered]
 
-      if updated_id_info[:entered] && updated_id_info[:entered] == 'true'
-        %i[country id_type id_number].each do |key|
+      if updated_id_info[:entered] == 'true'
```

Comment:
I don't think this condition `(entered == 'true')` is necessary for job type 6, but I'll defer to @beaglebets. If it's not necessary, consider moving the  JobType::DOCUMENT_VERIFICATION check above entered.